boa/src/builtins/math/mod.rs

```diff
@@ -19,542 +19,550 @@ use crate::{
     exec::Interpreter,
     BoaProfiler,
 };
-use rand::random;
 use std::f64;
 
 #[cfg(test)]
 mod tests;
 
-/// Get the absolute value of a number.
-///
-/// More information:
-///  - [ECMAScript reference][spec]
-///  - [MDN documentation][mdn]
-///
-/// [spec]: https://tc39.es/ecma262/#sec-math.abs
-/// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs
-pub fn abs(_: &mut Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
-    Ok(Value::from(if args.is_empty() {
-        f64::NAN
-    } else {
-        f64::from(args.get(0).expect("Could not get argument")).abs()
-    }))
-}
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub(crate) struct Math;
+
+impl Math {
+    /// Get the absolute value of a number.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-math.abs
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs
+    pub(crate) fn abs(_: &mut Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
+        Ok(Value::from(if args.is_empty() {
+            f64::NAN
+        } else {
+            f64::from(args.get(0).expect("Could not get argument")).abs()
+        }))
```

Comment:
I think this is more clear, right? We also avoid one panic path:
```suggestion
        Ok(args.get(0).map(|val| f64::from(val).abs()).unwrap_or(f64::NAN).into())
```

Reply:
Hmmm. why don't we use `map_or` and `map_or_else`

so this would be:
```rust
Ok(args.get(0).map_or(f64::NAN, |val| f64::from(val).abs()).into())
```
I think in this case its more readable than the first and second option.

What do you think? @Razican

Reply:
I think this would be some good `good fist issue`s

Reply:
Yes, these functions are more clear :)